webapp/apifuncs.py

```diff
@@ -243,12 +249,17 @@ def add_todays_batting(result1, result2, playercode, fullname, pos, clubname):
     # column heading will reflect AVG or AB based on which stats are displayed
     if 'batting' in result1.keys():
         game_stats = collate_stats(result1, game_stats)
-    if 'batting' in result1.keys():
+    if 'batting' in result2.keys():
```

Comment:
Everything looks good except for this small miss. The IF statement was updated to check for 'batting' in result2 for the twi-night double header game, but the function call executed is still passing result1. Address and I see an approval/merge in the near future.